src/themes/default.css

```diff
@@ -4,14 +4,30 @@
   /* Colors */
   --gr-color-primary: #f27a1a;
   --gr-color-primary-hover: #ef6114;
+  --gr-color-primary-outline: #FFF;
   --gr-color-secondary: #273142;
   --gr-color-secondary-hover: #0f131a;
-  --gr-color-success: #0bc15c;
-  --gr-color-success-hover: #09a44e;
-  --gr-color-content-primary: #273142;
-  --gr-color-content-secondary: #6e7787;
-  --gr-color-content-tertiary: #95a1b5;
-  --gr-color-content-passive: #afbbca;
+  --gr-color-secondary-outline: #FFF;
+  --gr-color-tertiary: #F1F2F7;
+  --gr-color-tertiary-hover: #D5D9E1;
+  --gr-color-tertiary-outline: #FFF;
+  --gr-color-success: #0BC15C;
+  --gr-color-success-hover: #09A44E;
+  --gr-color-success-outline: #FFF;
+  --gr-color-error: #FF5043;
+  --gr-color-error-hover: #FF3028;
+  --gr-color-error-outline: #FFF;
+  --gr-color-content-primary: #FFF;
+  --gr-color-content-outline-primary: #f27a1a;
+  --gr-color-content-secondary: #FFF;
+  --gr-color-content-outline-secondary: #273142;
+  --gr-color-content-tertiary: #273142;
```

Comment:
Font colors are always either white or same as the variant color, except tertiary. Let’s discuss this inconsistency in tertiary button with our design team. If we can align tertiary with others, we should not need content color definitions for buttons.

Reply:
Yeah, it is known issue by me. I already contacted to design team. For now, I will extract tertiary button for the design system's sake. You can check it in the next commit.

Reply:
You can follow this [comment](https://github.com/Trendyol/grace/pull/118#issuecomment-1146889597) for latest news about **tertiary** button.